models/staging/edfi_3/stage/_edfi_3__stage.yml

```diff
@@ -810,9 +810,6 @@ models:
       - name: k_program_evaluation
         tests:
           *ref_k_program_evaluation
-      - name: k_program
```

Comment:
So, we're expecting that there will be programs in `stg_ef3__student_program_evaluations` that aren't found in `stg__programs`?

Reply:
I would think so. Looks like I didn't select `v:programReference` in `base_ef3__student_program_evaluations` either, so that should be added too.

Reply:
Ope, I was looking at programEvaluations, not studentProgramEvaluations. No program reference there.